src/__tests__/inventory/view/ViewInventoryTable.test.tsx

```diff
@@ -27,53 +31,83 @@ describe('ViewInventoryTable', () => {
     id: 'someId',
     name: 'someName',
     rank: 'someRank',
-  } as Inventory
+    type: 'clothing',
```

Comment:
stylistically is there a reason why we are putting "some" in some fields and not in others?